infrastructure/variables.tf

```
@@ -1,4 +1,6 @@
-variable "product" {}
+variable "product" {
+  default = "unspec"
+}
```

don't set a default, it's set by the jenkins-library and these files often get copied around and out of sync
```suggestion
variable "product" {}
```